### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by bahlo**
- Contains the fix for the timeout setting (#8)
- Contains the fix for headers not being sent (#9)

Thanks @stayallive for the contributions!

It also bumps the version to 0.3.0, removes the keepalive parameter and does a minor change in the stringify logic.

### CODE REVIEW

File: src/worker.js [hunk 1]
```diff
@@ -48,8 +51,7 @@ async function sendLogs () {
   return fetch(url, {
     signal: AbortSignal.timeout(30_000),
     method: 'POST',
-    body: logs.map(JSON.stringify).join('\n'),
-    keepalive: true,
```
Comment (bdsqqq) [inline src/worker.js:52]: No need to keepalive anymore? 🤔 
just double checking

File: src/worker.js [hunk 2]
```diff
@@ -61,6 +63,20 @@ async function sendLogs () {
...
+
+  return [...headers].reduce((acc, [headerKey, headerValue]) => {
+    if (allowlist.includes(headerKey)) {
+      acc[headerKey] = headerValue;
+    }
+
+    return acc;
+  }, {});
+}
+
```
Comment (bdsqqq) [inline src/worker.js:79]: 👨‍🍳🤌

File: src/worker.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 const axiomDataset = 'my-dataset' // Your Axiom dataset
 const axiomToken = 'xapt-xxx' // Your Axiom API token
 
+const requestHeadersToCapture = ['user-agent'];
+const responseHeadersToCapture = ['cf-cache-status', 'cf-ray'];
```
Comment (bdsqqq) [inline src/worker.js:5]: Any relevant links to where we get these from?

**Review by bdsqqq — APPROVED** [review]

File: src/worker.js (hunk 1 above)
Comment (bahlo) [reply src/worker.js:52]: it's not used by cf anyway

File: src/worker.js (hunk 3 above)
Comment (bahlo) [reply src/worker.js:5]: those are the most useful, if there's specific others they can always be logged to console while developing

**Pull request merged**